src/main/java/com/spartronics4915/frc2020/subsystems/Launcher.java

```diff
@@ -0,0 +1,157 @@
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+public class Launcher extends SpartronicsSubsystem {
+    private CANSparkMax mFlywheelMasterMotor;
+    private CANSparkMax mFlywheelFollowerMotor;
+    private CANEncoder mFlywheelEncoder;
+    private SpartronicsSRX mAngleAdjusterMotor;
+    private SpartronicsEncoder mAngleAdjusterEncoder;
+    private SpartronicsSRX mTurretMotor;
+    private SpartronicsEncoder mTurretEncoder;
+    private double targetRPM;
+    private double targetAngle;
+    public Launcher() {
+        // Construct your hardware here
+        boolean success=false;
+        try {
+            //Two NEOs for flywheel (Master and follower, opposite directions)
+            mFlywheelMasterMotor = new CANSparkMax(Constants.kLauncherFlywheelMasterID,MotorType.kBrushless);
+            mFlywheelFollowerMotor = new CANSparkMax(Constants.kLauncherFlywheelFollowerID,MotorType.kBrushless);
+            mFlywheelFollowerMotor.follow(mFlywheelMasterMotor, true);
+            mFlywheelEncoder = mFlywheelMasterMotor.getEncoder();
+            //One snowblower for angle adjustement
+            mAngleAdjusterMotor = new SpartronicsSRX(Constants.kLauncherAngleAdjusterID,null);
+            mAngleAdjusterEncoder = mAngleAdjusterMotor.getEncoder();
+            //One BAG motor for turret
+            mTurretMotor = new SpartronicsSRX(Constants.kLauncherTurretID,null);
+            mTurretEncoder = mTurretMotor.getEncoder();
+            success = true;
+        } catch (Exception e) {
+            //TODO: handle exception
+            success = false;
+            logException("Could not instantiate Launcher: ", e);
+        }
+        logInitialized(success);
+    }
+
+    // Outline your API here by creating specific methods.
+    // Each method should perform a _singular action_
+    // - eg. instead of a setIntake method, control each intake motor individually
+    // setIntake functionality should be implemented in a command.
+    
+    /**
+     * @param relativeAngle Angle in degrees you want to turn the turret relative to the current angle
+     */
+    public void turnTurret(double relativeAngle) {
+        //rotates turret a specific angle relative to its current angle
+    }
+    /**
+     * @return Current angle in degrees the turret is facing relative to the home position (forwards)
+     */
+    public double getTurretDirection() {
+        //returns the current angle the turret is facing relative to straight ahead/home position
+        return mTurretEncoder.getPosition();
+    }
+    /**
+     * @param angle Angle in degrees above horizontal you want the angle adjuster to go to
+     */
+    public void setAngle(double angle) {
+        //sets target angle to given angle
+        targetAngle=angle;
+    }
+
+    /**
+     * @param rpm RPM you want the flywheel to target
+     */
+    public void setRPM(double rpm) {
+        //sets target rpm for flywheel to given rpm
+        targetRPM=rpm;
```

Comment:
@ethanrini this assumes there is a `default command` to act on changes to the RPM. If so, the change could be time dependent, as it will be based on when the `scheduler` gets to it. If this needs to be instantaneous, it will be better to use the `// TODO` or `// FIXME` syntax to remind yourself to set the motor RPM value directly.